src/app/backend-api.service.ts

```diff
@@ -963,7 +964,27 @@ export class BackendApiService {
             message: MessageText,
           })
           .pipe(
-            switchMap((encrypted) => {
+            tap({
+              next: (response: any) => {
+                if (response?.requestDerivedCookieWithEncryptedSeed) {
+                  this.identityService.launch('/log-in', {
+                    hideJumio: true,
+                    requestDerivedCookieWithEncryptedSeed:
+                      !!response.requestDerivedCookieWithEncryptedSeed,
+                  });
+                  return throwError(response); // end the execution of this call with an error
```

Comment:
Can we wait for the callback from the window API and then proceed if successful instead of throwing an error. Basically only throw the error if we don't have the cookie after the window API is launched?

Applies to all pipes below